daisy_workflows/image_import/inspection/src/boot_inspect/cli.py

```diff
@@ -45,6 +51,24 @@ def _output_human(results: model.InspectionResults):
   _output_json(results, indent=4)
 
 
+def _inspect_uefi(device):
+  try:
+    os.system('fdisk -l {} -o type'.format(device))
+    stream = os.popen('fdisk -l {} -o type'.format(device))
+    output = stream.read()
```

Comment:
The modern way of starting a process is `subprocess.run` -- it handles stdout, stderr, and the process's return code. 

I *think* the debian 9 worker has Python 3.5: https://docs.python.org/3.5/library/subprocess.html

(`ps.open` here drops stderr and ignores the return code)

Also, you've got a duplicate call to fdisk :)